.circleci/config.yml

```diff
@@ -0,0 +1,42 @@
+version: 2
+jobs:
+  unit:
+    docker:
+      - image: hashicorpdev/consul-helm-test:0.2.0
+
+    steps:
+      - checkout
+
+      - run:
+          name: Run Unit Tests
+          command: bats ./test/unit
+
+  acceptance:
+    docker:
+      - image: hashicorpdev/consul-helm-test:0.2.0
+
+    steps:
+      - checkout
+
+      - run:
+          name: Run Acceptance Tests
+          command: |
+            set -e
+
+            terraform init
+            terraform apply -var 'init_cli=true' -var 'project=${GOOGLE_PROJECT} --auto-approve
+
+            bats ./test/acceptance
+
+            terraform destroy -var 'project=${GOOGLE_PROJECT} --force
```

Comment:
I think there's a way to have this always run, regardless of pass/fail. Also `'` is unbalanced

Reply:
https://circleci.com/docs/2.0/configuration-reference/#the-when-attribute you can use `when:` inside a run step. In this case, the destroy can just be a separate `run` key so the when attribute is only applied to destroy and not everything else.

Reply:
I've split out the mono-step into multiple steps, and now `terraform destroy` can run always. Thank you both 😃